tests/unit_tests/test_mesh_builder.py

```diff
@@ -1,22 +1,22 @@
 
 import unittest
 import json
-from polar_route.mesh_generation.environment_mesh import EnvironmentMesh
-from polar_route.mesh_generation.mesh_builder import MeshBuilder
-from polar_route.mesh_generation.direction import Direction
+from meshiphi.mesh_generation.environment_mesh import EnvironmentMesh
+from meshiphi.mesh_generation.mesh_builder import MeshBuilder
+from meshiphi.mesh_generation.direction import Direction
 
-from polar_route.mesh_generation.boundary import Boundary
+from meshiphi.mesh_generation.boundary import Boundary
 class TestMeshBuilder(unittest.TestCase):
    def setUp(self):
       self.config = None
       self.env_mesh = None
       self.json_file = "../unit_tests/resources/global_grf_normal.json"
       with open (self.json_file , "r") as config_file:
           self.json_file = json.load(config_file)
-          self.config = self.json_file ['config']
+          self.config = self.json_file ['config']['mesh_info']
           self.mesh_builder =  MeshBuilder(self.config)
           self.env_mesh = self.mesh_builder.build_environmental_mesh()
-          self.env_mesh.save("global_mesh.json")
+         #  self.env_mesh.save("global_mesh.json")
```

Comment:
I guess this can just be deleted rather than left in as a comment?

Reply:
Ok, happy for this to stay as is if you're planning to look at it on another branch :+1: